bgc_data/bgc_lfish_samples.dataresource.yaml

```diff
@@ -30,6 +30,13 @@ schema:
     # https://docs.python.org/2/library/datetime.html#strftime-strptime-behavior
     format: "%Y-%m-%d %H:%M:%S"
     required: true
+  - name: SAMPLEDATEUTC
+    title: "Date & time of sample in local time (formatted string)"
+    type: datetime
+    # format string *must* be specified using standard Python/C strptime specification
+    # https://docs.python.org/2/library/datetime.html#strftime-strptime-behavior
+    format: "%Y-%m-%d %H:%M:%S"
+    required: true    
```

Comment:
Claire said
> It might be null if the tows go out of Australian water because then my code will fail, so best to allow this.

... so it should not be required.

Reply:
Thanks Marty - made that change now.